core/src/main/scala/kafka/tools/ConsumerPerformance.scala

```diff
@@ -213,13 +213,14 @@ object ConsumerPerformance {
 
     CommandLineUtils.checkRequiredArgs(parser, options, topicOpt, numMessagesOpt)
 
-    val useNewConsumer = options.has(useNewConsumerOpt)
+    val useOldConsumer = options.has(zkConnectOpt)
 
     val props = if (options.has(consumerConfigOpt))
       Utils.loadProps(options.valueOf(consumerConfigOpt))
     else
       new Properties
-    if (useNewConsumer) {
+    if(!useOldConsumer) {
```

Comment:
Also, we should probably check that `zkConnect` is not used together with `--new-consumer`.